requirements-integration.in

```
@@ -6,6 +6,7 @@ lightkube
 pytest-operator
 pyyaml
 requests
-selenium
+# Pinning to 4.10.x because upgrading to 4.11.2 results in chrome version compatibility issue. See https://github.com/canonical/dex-auth-operator/issues/151.
+selenium==4.10
```

Let's keep this a bit loose on the lower bound in case other packages (iff there are others) require selenium less than 4.10:

```suggestion
selenium<4.11
```